scene/gui/line_edit.h

```diff
@@ -58,6 +58,17 @@ class LineEdit : public Control {
 
 	};
 
+	enum VirtualKeyboardInputType {
+		KEYBOARD_TYPE_DEFAULT,
+		KEYBOARD_TYPE_MULTILINE,
+		KEYBOARD_TYPE_NUMBER,
+		KEYBOARD_TYPE_NUMBER_DECIMAL,
+		KEYBOARD_TYPE_PHONE,
+		KEYBOARD_TYPE_EMAIL_ADDRESS,
+		KEYBOARD_TYPE_PASSWORD,
+		KEYBOARD_TYPE_URL
+	};
```

Comment:
Is it possible to refer to the enum defined in `os.h` instead of redefining a new one here?

Reply:
I've seen this pattern used elsewhere in the engine, such as `Viewport::Usage` <-> `VisualServer::ViewportUsage`.
There is a bit of an interdependency between `OS`, `core_bind.h::_OS`, and `LineEdit`. To keep bindings straightforward, I think this is fine.